build/tutorials/tutorials-contest/hardhat-fork/README.md

```diff
@@ -0,0 +1,574 @@
+## Introduction
+
+In this tutorial we will cover the use of Hardhat, a powerful EVM development tool, together with Avalanche. More
+specifically, we will cover how to configure it to work with Avalanche CChain and how to use the Hardhat `fork`
+mechanism to test your DeFi dApps.
+
+If you want to jump right into the code, here's a [link](https://github.com/tbrunain/avalanche-hardhat-fork-tutorial)
+to the github repo containing the code of this tutorial.
+
+I would recommend you to clone the repo and start with the following commands:
+- npm i
+- npm test
+
+
+
+### Avalanche
+
+Avalanche is a platform for creating custom blockchain networks and decentralized applications (dApps). Created by Ava
```

Comment:
Since this is eventually going to be hosted on our docs site, you can cut most of this section as it won't be necessary.

Reply:
Done (Sorry did not receive any notifications from your github about your review : ( )